### DISCUSSION TITLE
TypeError: Cannot read property 'startsWith' of undefined

### DISCUSSION THREAD

**Discussion by himanshukp1** [nuxt/image#800] [General]
When i install 0.7.1 version, its throwing this error in my console when I am trying to run the application.. 0.6.2 is not throwing any error.

![image](https://user-images.githubusercontent.com/92923875/229544885-24810f76-2077-47f2-9572-5a1413d4515b.png)

**Comment by plushdohn** [comment]
For everyone who comes across this, in my case this error occurred when I set something that can be `undefined` in the `domains` array in `nuxt.config`, specifically I was whitelisting an environment variable like this:
```ts
domains: [process.env.SUPABASE_URL]
``` 
and in CI I did not set that env variable
